app/api/chemotion/reaction_api.rb

```diff
@@ -164,6 +164,8 @@ class ReactionAPI < Grape::API
         optional :segments, type: Array
         optional :variations, type: [Hash]
         optional :vessel_size, type: Hash
+        optional :gaseous, type: Boolean
+        optional :vessel_size, type: Hash
```

Comment:
It looks like there are duplicate changes from rebasing `reaction-scheme-adaptation` onto `main` (with `main` containing #2017). See for example this duplicate of `:vessel_size`.

Reply:
fixed